tutorials/using-slurm-to-host-jupyter-notebooks/startup-script.patch

```diff
@@ -0,0 +1,52 @@
+--- /tmp/slurm-gcp/scripts/startup-script.py    2018-12-10 10:41:30.255837647 -0500
```

Comment:
I will reiterate here that it may be better to get the original fixed, rather than running this patch.

Reply:
agreed. working on it